backend/tournesol/admin.py

```diff
@@ -143,7 +143,17 @@ def get_poll_name(self, obj):
 
 @admin.register(ContributorRatingCriteriaScore)
 class ContributorRatingCriteriaScoreAdmin(admin.ModelAdmin):
-    pass
+    list_filter = (
+        "contributor_rating__poll__name",
+    )
+    list_display = (
```

Comment:
Here too we are now not displaying the related entity.

See my comment about the `list_display` of `ComparisonCriteriaScoreAdmin`.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/794/commits/d226c0df727c390b8145c0e20af618d376e62c5d.